backend-api/infra/monitoring/alarms.yaml

```
@@ -1904,6 +1904,29 @@ Resources:
             - Key: MessageCode
               Value: $.messageCode
 
+  JourneyStartedMetricFilter:
```

Nitpick for consistency
```suggestion
  JourneysStartedMetricFilter:
```